tracer/build/_build/Build.Steps.cs

```diff
@@ -630,6 +630,42 @@ async Task DownloadWafVersion(string libddwafVersion = null, string uncompressFo
                     }
                 });
 
+    Target CopyNativeFilesForTests => _ => _
+        .Unlisted()
+        .After(Clean)
+        .After(BuildTracerHome)
+        .Executes(() =>
+        {
+            foreach(var projectName in Projects.NativeFilesDependentTests)
+            {
+                var project = Solution.GetProject(projectName);
+                var testDir = project.Directory;
+                var frameworks = project.GetTargetFrameworks();
+                var testBinFolder = testDir / "bin" / BuildConfiguration;
+
+                if (IsWin)
+                {
+                    foreach (var fmk in frameworks)
```

Comment:
[coding style nit-picking] We don't usually shorten works like this. Consider using `framework` instead of `fmk`. (also below in line 660)